src/main/java/edu/ucsb/cs156/happiercows/entities/Profit.java

```diff
@@ -0,0 +1,32 @@
+package edu.ucsb.cs156.happiercows.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "profit")
+public class Profit {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  @ManyToOne
+  @JoinColumn(name = "user_commons_id")
+  // Allows multiple profit entries in the table to be associated with a single user_commons_id
+
+  private UserCommons userCommons;
+  private long profit;
+  private String time;
```

Comment:
Is is better to use a String time here?  

Or a `LocalDateTime` as in this code from STARTER-team03?

https://github.com/ucsb-cs156-w22/STARTER-team03/blob/a357367a804aa3fc4290a01bd2309a91a2b41c0d/src/main/java/edu/ucsb/cs156/example/entities/UCSBDate.java#L27

Reply:
I was planning to use a library like [Day.js](https://day.js.org/) to parse the String time in the frontend, but if it is better to take care of that sort of thing here then I'll get working on it.

Reply:
So the front end is sort of a separate issue... here what's at issue is "how is the data stored in the database".

Storing it as a LocalDateTime should make it easier to do queries, sorting, etc. based on time.